metricbeat/scripts/mage/docsCollector.go

```diff
@@ -0,0 +1,488 @@
+package mage
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/elastic/beats/dev-tools/mage"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+// this struct provides module-level data that will be used to populate the module list
```

Comment:
godoc strings should start with the name of the thing being documented
```suggestion
// moduleData provides module-level data that will be used to populate the module list
```